js/ooyala_ssai.js

```diff
@@ -519,15 +534,22 @@ OO.Ads.manager(function()
     };
 
     /**
-     * Registered as a callback with the AMC, which gets called by the Ad Manager Controller when the replay button is
-     * clicked. Here it will try to load the rest of the vast ads at this point if there any.
-     * @public
-     * @method OoyalaSsai#onReplay
-     */
-    this.onReplay = () => {
-      currentOffset = 0;
-      this.currentAd = null;
-    };
+    *
+    *
+    */
+    var _getAdDuration = _.bind(function(id3Object)
```

Comment:
Please use an arrow function and remove `bind`
```
var _getAdDuration = (id3Object) => {
}
```

Reply:
Incorporated the changes.